content/articles/how-to-format-form-data-as-json/index.md

```diff
@@ -62,36 +65,42 @@ It’s assumed you have prior experience with HTML, CSS and JS.
 </form>
 ```
 
-The sample form we will be working with.
 ![The HTML form](sample-form.png)
```

Comment:
This image link isn't properly formatted.
Here's a proper example: "! [Simulator-interface](/engineering- education/implementing-africa-talking-sms-api-using-nodejs/simulator-interface.png)". Correct this for all image instances.